src/ui/server/controllers/insurance/business/broker/index.test.ts

```diff
@@ -82,6 +84,11 @@ describe('controllers/insurance/business/broker', () => {
             ID: EMAIL,
             ...BROKER_FIELDS[EMAIL],
           },
+          DETAILS: {
+            ID: DETAILS,
+            LINK: BROKER_LINK,
```

Comment:
No need for this, `LINKS` is already accessible in the page from `corePageVariables`, can just do this in the template:

```js
CONTENT_STRINGS.LINKS.EXTERNAL.APPROVED_BROKER_LIST
```